.travis.yml

```diff
@@ -1,10 +1,8 @@
-dist: trusty
+dist: xenial
```

Comment:
Since this package requires `php-7.4` version test, it's fine to run `xenial` dist on Travis CI building.